### PR TITLE
Merge liberty-ls-prototype branch into main

### PR DESCRIPTION
**Pull request by TrevCraw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
 # Liberty Tools for VS Code
 
-[![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/Open-Liberty.liberty-dev-vscode-ext.svg "Current Release")](https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext)
+## Building the Liberty Language Server Prototype
```
Comment (kathrynkodama) [inline README.md:3]: Do we want to keep this on the main README? Maybe it should be moved to a separate doc. Keep in mind that the README is used on the VS Code Marketplace extension page

File: src/extension.ts [hunk 2]
```diff
@@ -9,66 +9,93 @@
...
+let languageClient: LanguageClient;
+const SUPPORTED_LANGUAGE_IDS = ["properties", "plaintext"];
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
-	
-	let projectProvider = ProjectProvider.getInstance();
-	if ( !projectProvider ) {
-		projectProvider = new ProjectProvider(context);
-		ProjectProvider.setInstance(projectProvider);
-	}
-
```
Comment (travis1111) [inline src/extension.ts:22]: this should there

File: README.md (hunk 1 above)
Comment (TrevCraw) [reply README.md:3]: When you say the README is used on the Marketplace extension page, do you mean it's using a direct link to the `main` branch README and not a static copy somewhere?

File: src/extension.ts (hunk 2 above)
Comment (TrevCraw) [reply src/extension.ts:22]: This code block was moved to the `registerCommands` function below. Can you create a code suggestion with the extra lines that are needed?

File: README.md (hunk 1 above)
Comment (kathrynkodama) [reply README.md:3]: I think it's a direct link...

**Review by kathrynkodama — APPROVED** [review]
LGTM, we can queue up the readme update for another PR

File: src/extension.ts (hunk 2 above)
Comment (TrevCraw) [reply src/extension.ts:22]: Updated in https://github.com/OpenLiberty/liberty-tools-vscode/pull/148

**Pull request merged**